extensions/logobuilders.py

```diff
@@ -274,21 +274,44 @@ async def on_raw_message_delete(self, _):
 
     @commands.Cog.listener()
     async def on_voice_state_update(self, member:discord.Member, before: discord.VoiceState, after: discord.VoiceState):
+        async def create_embed(description: str):
+            embed = discord.Embed(
+                description=description, color=discord.Color.light_gray()
+            )
+            embed.set_author(name=member, icon_url=member.avatar.url)
+            await self.voice_logs_channel.send(embed=embed)
+
+        if isinstance(after.channel, discord.StageChannel) or isinstance(
+            before.channel, discord.StageChannel
+        ):
+            description = ""
```

Comment:
This is redundant. So is actually every other `description =` variable initialization, I'd just call `await create_embed("description here")`